index.js

```diff
@@ -18,6 +18,7 @@ function readFile() {
     path.join(homeDir, ".bash_history"),
     path.join(homeDir, ".zsh_history"),
     path.join(homeDir, ".history"),
+    path.join(homdir,".local/share/fish/fish_history"),
```

Comment:
Hi, Christopher!
We will test this today, and use a "try" and "catch" just in case this fails on other systems.